app/components/ui/Navigation/CategoryDropdown.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { ServiceCategory } from "pages/constants";
+import { useMenuToggle } from "../../../hooks/MenuHooks";
+import DropdownSubmenu from "./DropdownSubmenu";
+
+export const CategoryDropdown = ({
+  categories,
+  resultsTitle,
+}: {
+  categories: Readonly<ServiceCategory[]>;
+  resultsTitle: string;
+}) => {
+  const { activeSubMenu, handleMenuToggle, menuRef } = useMenuToggle();
```

Comment:
Could you invoke this hook directly in the `DropdownSubmenu` component? I don't understand why these have to be passed in from the outside component since this state and handler don't actually get used outside. It seems like all you need to do is pass in the links and a title? But I may be missing something 😅 

Oh, but if the logic of the hook was invoked inside the component, then you wouldn't even need the hook.